pom.xml

```diff
@@ -74,7 +79,6 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <sonatypeOssDistMgmtSnapshotsUrl>https://oss.sonatype.org/content/repositories/snapshots/</sonatypeOssDistMgmtSnapshotsUrl>
```

Comment:
All the sonatype distribution stuff should be removed. Not applicable for Apache.